grafana: update to 8.3.3

lotheac opened the pull request:
switch to using prebuilt js files, dropping the node build requirement.
also fix the baked-in version numbers in the binaries.

#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64 (musl)

lotheac commented:
ping @the-maldridge

the-maldridge commented:
I saw this and was quite surprised to see that the node version requirements had changed.  Originally this package was built to rebuild the JS locally, but the grafana project also provided precompiled versions.  Is that still the case?

lotheac commented:
I'm not sure, I didn't look very closely at versions greater than 7.3.8 since my immediate problem at the time only required 7.2.something, and since 7.4.0 would have meant bigger changes. To clarify, this PR does not change node version requirements.

the-maldridge commented:
Right, I'm asking if the node requirement can be removed completely.

lotheac commented:
Good question. I experimentally removed the yarn build steps from the package, but that resulted in the application returning a 500 error when installed. It would need further investigation. I can dig deeper a bit later -- do you want me to do so as part of this PR or separately?

the-maldridge commented:
Please do so as part of this PR.  The yarn build comes from a time where we insisted on building every part of a package internally, and while this is good in principal, for vendored JS it winds up being *a lot* more trouble than its worth.

lotheac commented:
rewrote commit to use upstream's prebuilt js files, removing the node requirement, and updated to latest available 8.3.3.

my old config file was incompatible with this version -- I'm not sure how things like that are handled in void.

lotheac commented:
seems like github broke PR builds today by disabling git:// cloning...

lotheac commented:
rebased since CI seems fixed on master.

the-maldridge commented:
Yes use of the legacy `git://` protocol is being phased out, we fixed this some time ago on master.  As to how config file changes are handled in void, the best answer I can give is "dubiously".  I will build and test after work today and look to getting this merged tonight.

lotheac commented:
the `gen-go` make target expects to run go code on the host machine, so cross compilation fails. the commit I added just now attempts to address it but doesn't (sorry -- shouldn't have pushed it without testing the cross compilation locally first). figuring it out.

lotheac commented:
cross builds will now (hopefully) succeed. apologies for making you approve the workflow so many times (:

the-maldridge commented:
Overall your patch looks okay.  I'm having trouble fetching from github at the moment, so the build isn't working locally.  Can you squash your commits in the meantime?

lotheac commented:
squashed.
